graylog2-web-interface/src/stores/users/LdapGroupsStore.ts

```diff
@@ -1,49 +1,47 @@
-/// <reference path="../../../declarations/jquery/jquery.d.ts" />
-
-declare var $: any;
-declare var jsRoutes: any;
+/// <reference path="../../../declarations/bluebird/bluebird.d.ts" />
 
+import jsRoutes = require('routing/jsRoutes');
+const fetch = require('logic/rest/FetchProvider').default;
 const UserNotification = require('util/UserNotification');
 const URLUtils = require('util/URLUtils');
 
+const LdapGroupsStore = {
+  loadGroups(): Promise<string[]> {
+    const promise = fetch('GET', URLUtils.qualifyUrl(jsRoutes.controllers.LdapController.apiGroups().url));
+    promise.catch(error => {
+      if (error.additional.status !== 404) {
```

Comment:
Good question, the check was already in the store and I simply adapted it. I don't see anything special in the backend code, so I think we can remove it.